packages/prerelease-registry/functions/utils/getArtifactForWorkflowRun.ts

```diff
@@ -66,7 +66,45 @@ export const getArtifactForWorkflowRun = async ({
 				{ status: 404 }
 			);
 
-		const zipResponse = await gitHubFetch(artifact.archive_download_url);
+		const redirResponse = await gitHubFetch(artifact.archive_download_url, {
```

Comment:
We are redirecting still, we’re just doing it manually below. The location header contains the redirect URL: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/302

We need to redirect manually so that we don’t include the Authorization header when redirecting. So this will fix all existing links

Reply:
Ah I see now, yes this is a valid concern. I know they delete artifacts after a while so they shouldn't be depended on, but I see a PR from 3 weeks ago has different behavior. I'll add backwards compatibility.
Great shout!

Reply:
Disregard that @mrbbot - my test was misconfigured - I confirmed that older links still work with this change.
Looks like they moved from `pipelinesghubeus2.actions.githubusercontent.com` to `productionresultssa8.blob.core.windows.net`. Neither of these domains require the Authorization header as their both pre-signed URLs in the query params.

So this PR is good to go as-is.

Reply:
Tested this change on PRs as old as October 25th and it works fine (PR artifacts much older than that have already been deleted by GitHub)

Reply:
Yeah retention is only 90 days